blueocean-dashboard/src/main/js/components/Runs.jsx

```diff
@@ -74,7 +75,9 @@ export default class Runs extends Component {
                 {id}
             </td>
             <td><CommitHash commitId={commitId} /></td>
-            {showBranchCol && <td>{decodeURIComponent(pipeline)}</td>}
+            <IfCapibility dataClass={pipelineClass} capability="io.jenkins.blueocean.rest.model.BlueMultiBranchPipeline" >
+                <td>{decodeURIComponent(pipeline)}</td>
+            </IfCapibility>
```

Comment:
I know this was what was suggested to you, but I'd be dead against us having Java class names (e.g. "io.jenkins.blueocean.rest.model.BlueMultiBranchPipeline") littered through the JavaScript code. Things like "isMultiBranch" etc should (imo) be encapsulated in some way, with class checks etc pushed down and out of site from the app code.